### ISSUE TITLE
Multiple cops fail inspecting Gemfile

### ISSUE THREAD

**Issue by sgapanovich** [rubocop-hq/rubocop-rspec#1085]
Multiple cops fail when inspecting a Gemfile 

`An error occurred while RSpec/Capybara/CurrentPathExpectation cop was inspecting ...RubymineProjects/ca_regression/Gemfile.`

It wasn't an issue a few days ago but I updated rubocop to 1.2.0 and then the issues appeared 

Mention the following information in the issue report:
1.2.0 (using Parser 2.7.2.0, rubocop-ast 1.1.1, running on ruby 2.6.6 x64-mingw32)

**Comment by pirj** [comment]
What is your version of `rubocop-rspec`? I suggest you update to the recently released 2.0, that should fix the issue.

**Comment by bquorning** [comment]
The stacktrace includes
```
C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/rubocop-rspec-2.0.0/lib/rubocop/cop/rspec/base.rb:22:in `on_new_investigation'
```
showing that RuboCop RSpec v2.0.0 is being used.

The error message ```undefined method `[]' for nil:NilClass``` in `lib/rubocop/cop/rspec/base.rb:22` suggests that either `config` or `config['RSpec']` is nil.

A lot of things don’t make sense. RuboCop RSpec shouldn’t even be inspecting Gemfile. 😕
